Palo Alto Networks/paloalto-ngfw/tests/fix_bug_without_int.json

```diff
@@ -41,14 +41,19 @@
         "name": "destuser"
       }
     },
+    "host": {
+      "name": "FWPA01"
+    },
     "log": {
+      "hostname": "FWPA01",
       "logger": "traffic"
     },
     "network": {
       "application": "windows-remote-management",
       "bytes": 2346,
       "packets": 9,
-      "transport": "tcp"
+      "transport": "tcp",
+      "type": "N/A"
```

Comment:
Can you add `N/A` in the `ignored_values` of the parser?

Reply:
Thank you, I didn't see it.